### PR TITLE
Allows giving a custom name to the Survey results page

### LINKED ISSUE #63: Cannot give a custom name to the Survey results page
I want to give the title 'DNN Survey Results' a custom name (not the standard name), but that look impossible. 
Am i missing something?

SnipImage (002)

### PR DESCRIPTION
**Pull request by MaiklT**
## Changes made
Added keys to Resource files for control titles and module help, added messages to controls to display a link to the documentation

## PR Template Checklist

- [x] Fixes Bug
- [ ] Feature solution
- [ ] Other

## Please mark which issue is solved
Close #63

### CODE REVIEW

File: DNN.Survey/App_LocalResources/SurveyResults.ascx.resx [hunk 1]
```diff
@@ -117,9 +117,15 @@
   <resheader name="writer">
     <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
   </resheader>
+  <data name="ControlTitle_surveyresults.text" xml:space="preserve">
+    <value>Ergebnisse</value>
```
Comment (valadas) [inline DNN.Survey/App_LocalResources/SurveyResults.ascx.resx]: This would should be english

**Review by valadas — CHANGES_REQUESTED** [review]
Thanks, just one piece of text is not english...

File: DNN.Survey/App_LocalResources/SurveyResults.ascx.resx (hunk 1 above)
Comment (MaiklT) [reply DNN.Survey/App_LocalResources/SurveyResults.ascx.resx]: I changed it and commited it directly to the pull request.

**Review by valadas — APPROVED** [review]
Awesome, thanks!

**Pull request merged**